### PR TITLE
Development

### PR DESCRIPTION
**Pull request by IgorKurkov**
Lets make big pull request for releasing something to master branch

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ import { ArchiveComponent } from './content/archive/archive.component';
 import {MatCardModule} from '@angular/material/card';
```
[the remark this answers is not in the document]
Comment (zophrox) [reply src/app/app.module.ts:18]: What do you mean? All project?

File: src/app/content/app-routing.module.ts [hunk 2]
```diff
@@ -7,11 +6,14 @@ import { HomeComponent } from './home/home.component';
 
 const appRoutes: Routes = [
   { path: '', component: HomeComponent },
-  // { path: 'competition', component: CompetitionRequlationsComponent },
-  { path: 'about_competition', component: AboutCompetitionComponent },
```
[the remark this answers is not in the document]
Comment (zophrox) [reply src/app/content/app-routing.module.ts:11]: ok, thank you

File: src/app/app.module.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (IgorKurkov) [reply src/app/app.module.ts:18]: Yes

File: src/app/content/app-routing.module.ts [hunk 3]
```diff
@@ -7,11 +6,14 @@ import { HomeComponent } from './home/home.component';
 
 const appRoutes: Routes = [
   { path: '', component: HomeComponent },
-  // { path: 'competition', component: CompetitionRequlationsComponent },
```
[the remark this answers is not in the document]
Comment (IgorKurkov) [reply src/app/content/app-routing.module.ts:10]: Yes

File: src/app/content/archive/about-competition/about-competition.component.html [hunk 4]
```diff
@@ -1,13 +1,27 @@
...
+  <h1>Молодша вікова категорія</h1>
+  <div *ngFor="let laureatInfo of laureatsInfo">
+    <app-card-laureat
+      *ngIf="laureatInfo.ageCategories === 'Молодша'"
+      [laureatInfo]="laureatInfo"
+    ></app-card-laureat>
+  </div>
+
+  <h1>Середня вікова категорія</h1>
+  <div *ngFor="let laureatInfo of laureatsInfo">
```
[the remark this answers is not in the document]
Comment (IgorKurkov) [reply src/app/content/archive/about-competition/about-competition.component.html:13]: Please take a look on comment above

File: src/app/content/archive/about-competition/about-competition.component.ts [hunk 5]
```diff
@@ -1,5 +1,8 @@
 import { Component, OnInit } from '@angular/core';
-import { CompetitionsInform } from 'src/app/shared/competitions-inform.model';
+import { ActivatedRoute, Params } from '@angular/router';
+
+// import { CompetitionsInform } from 'src/app/shared/competitions-inform.model';
```
[the remark this answers is not in the document]
Comment (IgorKurkov) [reply src/app/content/archive/about-competition/about-competition.component.ts:4]: Also

**Pull request merged**